src/localplayer.h

```diff
@@ -145,7 +145,7 @@ class LocalPlayer : public Player
 
 private:
 	// clang-format off
-	void accelerateHorizontal(const v3f &target_speed, f32 max_increase, bool slippery);
+	void accelerateHorizontal(const v3f &target_speed, f32 max_increase);
```

Comment:
Also remove the function param in the .cpp file, including all related calculations.